src/components/base/GhostButton.styles.ts

```diff
@@ -64,4 +64,8 @@ export const Button = styled.button<{ $reverse: boolean; $size?: 'sm'; $colored?
             cursor: not-allowed;
           }
         `}
+
+  &:focus:not(:focus-visible) {
```

Comment:
ok plusieurs occurences, je veux bien rediscuter du pb ?

Reply:
https://css-tricks.com/the-focus-visible-trick/